contrib/internal/httptrace/config.go

```diff
@@ -47,6 +54,7 @@ func newConfig() config {
 	}
 	v := os.Getenv(envServerErrorStatuses)
 	if fn := GetErrorCodesFromInput(v); fn != nil {
+		fmt.Println("MTOFF: is 200 error?", fn(200))
```

Comment:
Test left behind? 😁

Reply:
The same question applies for `contrib/internal/httptrace/httptrace.go`.

Reply:
Yes! Sorry, still in draft, but thanks for taking a look nonetheless 🙇